src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseConfirmsNotToProceedRespondentNotificationHandler.java

```diff
@@ -58,6 +64,10 @@ private CallbackResponse notifyRespondentSolicitorForClaimantConfirmsNotToProcee
             ? caseData.getApplicantSolicitor1UserDetails().getEmail()
             : caseData.getRespondentSolicitor1EmailAddress();
 
+        if (isRespondentSolicitor2Notification(callbackParams)) {
```

Comment:
We have called the method once above in line 50. Maybe store it and use the result here.

Reply:
calls to this method are not made in uniform sequence. hence it would fail and the flag will resolve to false in one of the calls if it was supposed to be true.